persistent/Database/Persist/Sql/Orphan/PersistQuery.hs

```diff
@@ -291,51 +295,52 @@ filterClauseHelper includeTable includeWhere conn orNull filters =
+                            (True, Eq, _) -> (T.concat
+                                [ name
+                                , " IS NULL"
+                                ],[])
+                            (True, Ne, _) -> (T.concat
+                                [ name
+                                , " IS NOT NULL"
+                                ],[])
                             (False, Ne, _) -> (T.concat
-                                [ "("
-                                , name
+                                [ name
                                 , " IS NULL OR "
                                 , name
                                 , " <> "
                                 , qmarks
-                                , ")"
                                 ], notNullVals)
                             -- We use 1=2 (and below 1=1) to avoid using TRUE and FALSE, since
                             -- not all databases support those words directly.
                             (_, In, 0) -> ("1=2" <> orNullSuffix, [])
                             (False, In, _) -> (name <> " IN " <> qmarks <> orNullSuffix, allVals)
                             (True, In, _) -> (T.concat
-                                [ "("
-                                , name
+                                [ name
                                 , " IS NULL OR "
                                 , name
                                 , " IN "
                                 , qmarks
-                                , ")"
                                 ], notNullVals)
                             (False, NotIn, 0) -> ("1=1", [])
                             (True, NotIn, 0) -> (name <> " IS NOT NULL", [])
                             (False, NotIn, _) -> (T.concat
-                                [ "("
-                                , name
+                                [ name
                                 , " IS NULL OR "
                                 , name
                                 , " NOT IN "
                                 , qmarks
-                                , ")"
                                 ], notNullVals)
                             (True, NotIn, _) -> (T.concat
-                                [ "("
-                                , name
+                                [ name
                                 , " IS NOT NULL AND "
                                 , name
                                 , " NOT IN "
                                 , qmarks
-                                , ")"
                                 ], notNullVals)
-                            _ -> (name <> showSqlFilter pfilter <> "?" <> orNullSuffix, allVals)
+                            _ -> (T.concat 
+                                [name
+                                   <> showSqlFilter pfilter <> "?" <> orNullSuffix
+                                ], allVals)
```

Comment:
these formatting changes make it a bit difficult to understand the signal v noise, can you back them out?

Reply:
yes!